File: Work/Old stuff/Basic Phyton/Exercises/Exercise 17.py
numlist = [1412,1255,1566,4598,1251,1532,1563,1245,1556,1345]
a = int(input("Enter a 4 digit number: "))
for i in numlist:
    if i == a:
        print("The number is in the list!")
        break
else:
    print("The number is not in the list.")

File: Work/Advanced C++/Exercises/Exercise17.cpp
#include <iostream>
#include <fstream>
using namespace std;
float Calc(float broj1, float broj2,string calcval){
    if(calcval == "*")
        return broj1*broj2;
    else if(calcval == "/")
        return broj1/broj2;
    else if(calcval == "+")
        return broj1+broj2;
    else if(calcval == "-")
        return broj1-broj2;
}
int main(){
    float a,b;
    string c;
    cout<<"Enter first number: ";
    cin>>a;
    cout<<"Enter second number: ";
    cin>>b;
    cout<<"Enter action(+,-,*,/): ";
    cin>>c;
    cout<<a<<c<<b<<"="<<Calc(a,b,c)<<endl;
}
